src/components/WalletConnect.tsx

```diff
@@ -0,0 +1,96 @@
+import { BrowserProvider } from "ethers";
+import log from "loglevel";
+import { createEffect, createResource } from "solid-js";
+
+import { config } from "../config";
+import { RBTC } from "../consts/Assets";
+import { useWeb3Signer } from "../context/Web3";
+import loader from "../lazy/walletConnect";
+import WalletConnectProvider from "../utils/WalletConnectProvider";
+
+const getLocation = () => {
+    const { protocol, host } = window.location;
+    return `${protocol}//${host}`;
+};
+
+export const WalletConnect = () => {
+    const { openWalletConnectModal, setOpenWalletConnectModal } =
+        useWeb3Signer();
+
+    const [createdKit] = createResource(async () => {
+        const configRsk = config.assets[RBTC];
+
+        const { appKit, EthersAdapter } = await loader.get();
+        return appKit.createAppKit({
+            themeMode: "dark",
+            enableEIP6963: false,
+            enableInjected: false,
+            adapters: [new EthersAdapter()],
+            projectId: import.meta.env.VITE_WALLETCONNECT_PROJECT_ID as string,
+            networks: [
+                {
+                    id: configRsk.network.chainId,
+                    name: configRsk.network.chainName,
+                    nativeCurrency: {
+                        name: RBTC,
+                        symbol: RBTC,
+                        decimals: 18,
+                    },
+                    rpcUrls: {
+                        default: {
+                            http: configRsk.network.rpcUrls,
+                        },
+                    },
+                    blockExplorers: {
+                        default: {
+                            name: "Explorer",
+                            url: configRsk.blockExplorerUrl.normal,
+                        },
+                    },
+                },
+            ],
+            metadata: {
+                name: "Boltz",
+                description: "Boltz Web App",
+                url: getLocation(),
+                icons: [`${getLocation()}/android-chrome-512x512.png`],
+            },
+            features: {
+                email: false,
+                socials: false,
+                analytics: true,
```

Comment:
What kinds of analytics is this about?

Reply:
https://docs.reown.com/appkit/react-native/cloud/analytics